charts/openshift-metering/templates/openshift-reporting/report-queries/pod-memory.yaml

```diff
@@ -362,22 +362,21 @@ spec:
     ), namespace_memory_request AS (
       {| renderReportQuery .Report.Inputs.NamespaceMemoryRequestQueryName . |}
     )
-    SELECT
-      timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
-      timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
-      usage.namespace,
-      usage.pod_usage_memory_byte_seconds,
-      request.pod_request_memory_byte_seconds,
-      usage.pod_usage_memory_byte_seconds / capacity.total_cluster_capacity_memory_byte_seconds AS pod_memory_usage_percent,
-      request.pod_request_memory_byte_seconds / capacity.total_cluster_capacity_memory_byte_seconds AS pod_memory_request_percent,
-      capacity.total_cluster_capacity_memory_byte_seconds
-    FROM namespace_memory_request as request
-    JOIN namespace_memory_usage as usage
-      ON request.period_start = usage.period_start
-      AND request.period_end = usage.period_end
-      AND request.namespace = usage.namespace
-    JOIN cluster_memory_capacity as capacity
-      ON capacity.period_start = request.period_start
-      AND capacity.period_end = request.period_end
-      AND capacity.period_start = usage.period_start
-      AND capacity.period_end = usage.period_end
+  SELECT
+    timestamp '2020-08-31 16:11:00.000' AS period_start,
+    timestamp '2020-08-31 16:52:00.000' AS period_end,
```

Comment:
Revert, this still needs to be a template.

Reply:
I'll template these two lines